src/Analyzer/Passes/UniqInjectiveFunctionsEliminationPass.cpp

```diff
@@ -41,9 +41,9 @@ class UniqInjectiveFunctionsEliminationVisitor : public InDepthQueryTreeVisitorW
             return;
 
         bool replaced_argument = false;
-        auto & uniq_function_arguments_nodes = function_node->getArguments().getNodes();
+        auto replaced_uniq_function_arguments_nodes = function_node->getArguments().getNodes();
```

Comment:
Why do you need to make a copy here?

Reply:
In case we end up discarding the optimization because of the return type change.